src/averageAges.js

```diff
@@ -14,12 +14,24 @@
  *
  * @return {number}
  */
+function getAverageDiff(diffs) {
+  const diffsSum = diffs.reduce((a, b) => a + b);
+
+  return diffsSum / diffs.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men;
+
+  century
+    ? men = people
+      .filter((p) => (p.sex === 'm' && Math.ceil(p.died / 100) === century))
+    : men = people
+      .filter((p) => (p.sex === 'm'));
```

Comment:
```suggestion
  const men = people.filter((p) => (p.sex === 'm' && century ? Math.ceil(p.died / 100) === century) : true)
```
the way you did it might be acceptable (except 1 thing*), but it would be more laconic to write it in this way

* not necessary to create `men` variable using let, you can do it in this way (correct usage of ternary operator):

```suggestion
    const men = century
      ? people.filter((p) => (p.sex === 'm' && Math.ceil(p.died / 100) === century))
      : people.filter((p) => (p.sex === 'm'));
```

Reply:
Thanks! Finally I get it about ternary :)